main/ertp/ERTP-Guide/PursesAndPayments.md

```diff
@@ -0,0 +1,269 @@
+an `amount` of 10 quatloos, the `payment` will always either have an
+`amount` of 10 quatloos or an `amount` of 0 quatloos, never anything
+in-between or larger than 10.
+
+Payments can be deposited in purses, split into multiple 
+payments, combined, and claimed (getting an exclusive payment and
+revoking access from anyone else). 
+
+Payments are often received from other actors and therefore should not
+be trusted themselves. To get the balance of a payment, use the
+`getAmountOf(payment)` method on the trusted `issuer` for the `brand`
+of the `payment`.  **tyg todo: How does one get said trusted issuer?**
+
+To convert a `payment` into a `purse`: 
+1. Access a trusted `issuer` for the `payment`'s `brand`.. 
+2. Create an empty purse with `issuer.makeEmptyPurse()`
+3. Put the payment `amount` in the purse with `purse.deposit(payment)`
+
+Payments have one API method, but many methods for other object
+types have `payment` objects as arguments and effectively operate on `payments`.
+- [`payment.getAllegedBrand()`](https://agoric.com/documentation/ertp/api/payment.html#payment-getallegedbrand)
+  - Returns the `payment`'s `brand`, indicating what digital asset the
+  payment purports to be. Note that `payment`s are not trusted, so this
+  result should be treated with suspicion and verified before 
+  use. **tyg todo: How do I verify?**
+
+Other objects' `payment`-related methods:
+
+- [`issuer.getAmountOfPayment(payment)`](https://agoric.com/documentation/ertp/api/issuer.html#issuer-getamountof-payment)
+  - Get the `payment`'s `amount`. The `payment` itself is not trusted,
+    so you must use this `issuer` method to be sure of getting the
+    true value. 
+  ` ```js
+	  const { issuer, mint, amountMath } = produceIssuer('bucks');
+	  const payment = mint.mintPayment(amountMath.make(100));
+	  issuer.getAmountOf(payment); // returns 100
+    ```
+- [`issuer.burn(payment, optAmount)`](https://agoric.com/documentation/ertp/api/issuer.html#issuer-burn-payment-optamount)
+  - Burn all of the digital assets in the `payment`. `optAmount` is
+    optional but if present, the `payment` balance must be equal to
+    it. If `payment` is a promise, the operation proceeds after it resolves. 
+  - ```js
+	  const { issuer, mint, amountMath } = produceIssuer('bucks');
+	  const amountToBurn = amountMath.make(10);
+	  const paymentToBurn = mint.mintPayment(amountToBurn);
+	  // burntAmount should equal 10
+	  const burntAmount = issuer.burn(paymentToBurn, amountToBurn);
+	  ```
+- [`issuer.claim(payment, optAmount)`](https://agoric.com/documentation/ertp/api/issuer.html#issuer-claim-payment-optamount)
+  - Transfer all assets from the `payment` to a returned new `payment`
+    and burn the original. No other references to this payment survive, so 
+    the new `payment`'s owner is the exclusive one. If `optAmount` is
+    present, `payment`'s balance is equal to `optAmount`'s, or it throsw
```

Comment:
```suggestion
    present, `payment`'s balance is equal to `optAmount`'s, or it throws
```
Could we have a spell check done before future reviews? I think that would save both of us a lot of time